app/src/main/java/com/codesoom/assignment/TaskManager.java

```diff
@@ -0,0 +1,62 @@
+package com.codesoom.assignment;
+
+
+import com.codesoom.assignment.Exception.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Task 목록에 대한 CRUD 처리를 담당합니다.
```

Comment:
이제 `목록`이라는 표현도 삭제해 봅시다. `Task`가 목록으로 관리되는지 집합으로 관리되는지, 트리로 관리되는지는 굳이 외부에 알리지 않아도 괜찮습니다.

```suggestion
 * Task에 대한 CRUD 처리를 담당합니다.
```

Reply:
저는 목록이라는 용어를 자료형보다는 일반적인 Task의 집합체 라는 개념으로 사용했는데, 
javadoc을 읽는 사람이 프로그래머이기 때문에 헷가릴 수 있겠네요.

Reply:
네. 프로그래밍에서는 목록(List)과 집합(Set)을 엄격히 나눕니다. 한편 수학에서 List는 Ordered Set(순서가 있는 집합) 으로, Set의 특수한 형태에요. 이것이 프로그래밍에도 전해지게 되었죠.